patch/patcher.go

```diff
@@ -531,12 +532,29 @@ func (p *Patcher) patchIdent(id *ast.Ident, obj types.Object) {
 			if v.Tag == nil {
 				v.Tag = &ast.BasicLit{}
 			}
-			v.Tag.Value = "`" + strings.TrimSpace(strings.Trim(v.Tag.Value, "` ")+" "+tags) + "`"
+
+			v.Tag.Value = "`" + mergeTags(v.Tag.Value, tags) + "`"
```

Comment:
Since `mergeTags` handles removing the outer "`" from the string, it should also handle adding them back.